backend/Dockerfile.visualization

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Google LLC
```

Comment:
maybe include a quick readme which has a two line synopsis of the overall flow.  dockerfile starts server.py which accepts a post request which ... 
makes it easy for the reader to zoom in on the most important parts in the least amount of time.